frameworks/g-cloud-12/questions/services/serviceDescription.yml

```diff
@@ -13,8 +13,8 @@ max_length_in_words: 50
 
 validations:
   - name: answer_required
-    message: You need to answer this question.
+    message: Enter summary.
   - name: under_50_words
-    message: Your description must be no more than 50 words.
+    message: Description must be 50 words or fewer.
   - name: under_character_limit
```

Comment:
Query: are there 2 separate limitations for this field - word and character count? Could we limit it to word count only?